onnx/utils.py

```diff
@@ -6,23 +6,22 @@
 import os
 import tarfile
 from collections import deque
-from typing import TYPE_CHECKING
 
 import onnx.checker
 import onnx.helper
 import onnx.shape_inference
 from onnx import FunctionProto, ModelProto, NodeProto, TensorProto, ValueInfoProto
 
-if TYPE_CHECKING:
-    from google.protobuf.internal.containers import RepeatedCompositeFieldContainer
-
 
 class Extractor:
     def __init__(self, model: ModelProto) -> None:
         self.model = model
         self.graph = self.model.graph
-        self.wmap = self._build_name2obj_dict(self.graph.initializer)
-        self.vimap = self._build_name2obj_dict(self.graph.value_info)
+        self.initializers = self._build_name2obj_dict(self.graph.initializer)
+        self.value_infos = self._build_name2obj_dict(self.graph.value_info)
```

Comment:
Optional: what about name like `initializer_dict`? I wonder if we may think this is a list.

Reply:
According to https://google.github.io/styleguide/pyguide.html#3161-names-to-avoid, `_dict` should be avoided. I added type annotations instead.